analysisPlugin/src/main/scala/org/scalaclean/analysis/ScalaCompilerPluginComponent.scala

```diff
@@ -498,13 +498,33 @@ class ScalaCompilerPluginComponent(val global: Global) extends PluginComponent w
 
         // *********************************************************************************************************
         case valDef: ValDef =>
-          val symbol = valDef.symbol
+          //its only a var due to for https://github.com/scala/bug/issues/12213 -- see below
+          var symbol = valDef.symbol
           val isVar  = symbol.isVar
           val fields: Option[ModelFields] = valDef.rhs match {
             case Select(qualifier, name) =>
               Option(qualifier.symbol).flatMap(_.attachments.get[ModelFields])
             case _ => None
           }
+          //workaround for https://github.com/scala/bug/issues/12213
+          if (!isVar || !valDef.rhs.isEmpty) valDef.pos.`end` else {
```

Comment:
Looks like you can drop `then` part, btw

Reply:
agreed, fixed